Cargo.toml

```diff
@@ -32,6 +32,7 @@ random = ["dep:rand"]
 yoga_benchmark = []
 debug = []
 profile = []
+taffy_tree = []
```

Comment:
Ohh, one more change: can you make the `slotmap` dependency optional and gate it behind the `taffy_tree` feature please?